projects/js-packages/publicize-components/src/components/social-previews/use-post-data.js

```diff
@@ -51,38 +51,35 @@ export function usePostData() {
 
 			const media = [];
 
-			// Attach media only if "Share as a social post" option is enabled.
-			if ( shouldUploadAttachedMedia ) {
-				if ( sigImageUrl ) {
-					media.push( {
-						type: 'image/jpeg',
-						url: sigImageUrl,
-						alt: '',
-					} );
-				} else {
-					const getMediaDetails = id => {
-						const mediaItem = getMedia( id );
-						if ( ! mediaItem ) {
-							return null;
-						}
-						return {
-							type: mediaItem.mime_type,
-							url: getMediaSourceUrl( mediaItem ),
-							alt: mediaItem.alt_text,
-						};
+			if ( sigImageUrl ) {
+				media.push( {
+					type: 'image/jpeg',
+					url: sigImageUrl,
+					alt: '',
+				} );
+			} else {
+				const getMediaDetails = id => {
+					const mediaItem = getMedia( id );
+					if ( ! mediaItem ) {
+						return null;
+					}
+					return {
+						type: mediaItem.mime_type,
+						url: getMediaSourceUrl( mediaItem ),
+						alt: mediaItem.alt_text,
 					};
+				};
 
-					for ( const { id } of attachedMedia ) {
-						const mediaDetails = getMediaDetails( id );
-						if ( mediaDetails ) {
-							media.push( mediaDetails );
-						}
+				for ( const { id } of attachedMedia ) {
+					const mediaDetails = getMediaDetails( id );
+					if ( mediaDetails ) {
+						media.push( mediaDetails );
 					}
-					if ( 0 === media.length && featuredImageId ) {
-						const mediaDetails = getMediaDetails( featuredImageId );
-						if ( mediaDetails ) {
-							media.push( mediaDetails );
-						}
+				}
+				if ( 0 === media.length && featuredImageId ) {
```

Comment:
I think because of this change we're now treating the featured image like attached media, so when a post has a featured image we're showing it in the preview like an attached image. I think, for now, we want that to be the link preview.

Reply:
Fixed in df7a9fa